src/bot/checks.rs

```diff
@@ -55,6 +56,34 @@ pub async fn is_admin(ctx: Context<'_>) -> bool {
     false
 }
 
+/// Returns None when there was an error,
+/// Some(true) when the user is a bot admin and
+/// Some(false) when the user is not a bot admin
+pub async fn is_bot_admin(ctx: Context<'_>) -> Option<bool> {
```

Comment:
I don't think None is very useful here as this should just be interpreted as false. The Methods calling this shouldn't actually care why a check returns false, only that the user doesn't have the permission to do so.